charts/alloy/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: alloy
+description: A Helm chart for Kubernetes
+type: application
+version: 0.1.0
+appVersion: 1.0.0
+
+dependencies:
+- name: alloy-api
+  version: "0.1.0"
+  repository: https://helm.cyberforce.site/charts
+- name: alloy-ui
+  version: "0.1.0"
+  repository: https://helm.cyberforce.site/charts
+- name: alloy-psql
```

Comment:
If I'm deploying this, I want to connect to my existing postgres server, not use another psql container/pvc.  IMO, db shouldn't be bundled with an app chart, rather document an available postgres chart if an operator wants to deploy it.